src/polyglot-notebooks-vscode-common/src/metadataUtilities.ts

```diff
@@ -127,6 +128,11 @@ export function getNotebookDocumentMetadataFromNotebookDocument(document: vscode
 
             const items = kernelInfo.items;
             if (Array.isArray(items) && items.every(item => typeof item === 'object')) {
+                items.forEach(ki => {
+                    if (ki.languageName === null || ki.languageName === undefined) {
+                        delete ki["languageName"];
```

Comment:
There are 3 places in this file where we do `delete foo["langugaeName"]`.  Let's collapse them into one.

Reply:
done